deploy/example/cloning/README.md

```diff
@@ -0,0 +1,67 @@
+# Volume Cloning Example
```

Comment:
also add this folder to https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/c59a0b75d2094e189fee72b367d543e7a9636674/hack/verify-yamllint.sh#L24

Reply:
done